specification/_global/info/examples/response/RootNodeInfoResponseExample2.yaml

```diff
@@ -0,0 +1,19 @@
+# summary: ''
+description: A successful response from `GET /` on Serverless.
```

Comment:
You can also add as much information to the description field as you like to provide context

Reply:
When the user clicks on the version OBJECT item:

<img width="519" height="40" alt="image" src="https://github.com/user-attachments/assets/f2c1d139-bcaf-4da3-af87-1cc5d2a666c5" />

Is it going to show something like the following?

```
  /**
   * The Elasticsearch version number.
   *
   * ::: IMPORTANT: For Serverless deployments, this static value is always `8.11.0` and is used solely for backward compatibility with legacy clients.
   *  Serverless environments are versionless and automatically upgraded, so this value can be safely ignored.
   */
```